webots_ros2_universal_robot/webots_ros2_universal_robot/follow_joint_trajectory_client.py

```diff
@@ -56,9 +58,12 @@ def __on_get_result_callback(self, future):
             rclpy.shutdown()
 
     def send_goal(self, trajectory, iteration=1):
-        self.get_logger().info('Waiting for action server...')
+        self.get_logger().info('Waiting for action server to be ready...')
         self.__client.wait_for_server()
 
+        sleep_time = 0.25
+        time.sleep(sleep_time)
```

Comment:
@lukicdarkoo just to be sure you also have those lines in your test without rendering ?

Reply:
Let me try without it